src/Templates/NIPSWeb/main.twig

```diff
@@ -4,39 +4,38 @@
 <link rel="stylesheet" type="text/css" href="{{baseurl}}css/planner.css" />
 {% endblock %}
 {% block content %}
-<div id="notice" class="ui-state-highlight ui-helper-hidden">Saving...</div>
+<div id="notice" class="alert alert-warning">Preparing show plan...</div>
 {# <div id="head-icons"><button class="btn-feedback">Send Feedback</button>&nbsp;<button class="btn-help"></button><button class="btn-logout" onclick="window.location = '/portal/redir_out.php'"></button></div> #}
 <header id="content-header">
   <h3 id="a-manage-library"><a href="{{myury.makeURL(config,'NIPSWeb','manage_library')}}" title="Library Manager">Manage Library</a></h3>
   <h2>{{title}}</h2>
 </header>
 <div id="grid" class="transBG clearfix">
-
-  <div id="bapswrapper" class="ui-widget">
+  <div id="bapswrapper">
     <div id="baps-channel-container" class="clearfix">
       {% for i in 0..2 %}
-        <ul class="box width-1 height-2 baps-channel ui-widget-content" channel="{{i+1}}" id="baps-channel-{{i + 1}}">
+        <ul class="box col-md-3 col-sm-3 col-xs-3 height-2 baps-channel" channel="{{i+1}}" id="baps-channel-{{i + 1}}">
```

Comment:
Ah, I added sm and xs because just md made smaller screens flatten them to full width.